### PR TITLE
Use ControlModule for getting the move direction

### PR DESCRIPTION
**Pull request by tacheometry**
This change allows for controller, mobile, keyboard, etc support. Directions are also analogue now, so move vectors such as "diagonally to the left" can be inputted.

### CODE REVIEW

**Comment by tacheometry** [comment]
One caveat is that some users might have forked the core scripts, and this method might not be available to them if these modules are deleted.

**Review by grilme99 — CHANGES_REQUESTED** [review]
This should ideally fall back to the original method if the ControlModule is unavailable. It would also be cleaner if getting the move vector is put into its own method.

File: src/Client/init.lua [hunk 1]
```diff
@@ -23,21 +28,12 @@ function ChickynoidClient:MakeCommand(deltaTime)
...
-                command.x = -1
-            elseif v.KeyCode == Enum.KeyCode.D then
-                command.x = 1
-            elseif v.KeyCode == Enum.KeyCode.Space then
-                command.y = 1
-            end
-        end
+    local moveVector = ControlModule:GetMoveVector()
+    if moveVector.Magnitude > 0 then
+        moveVector = moveVector.Unit
```
Comment (OverHash) [inline src/Client/init.lua:33]: Doesn't `ControlModule:GetMoveVector()` already return a unit vector?

Comment (tacheometry) [reply src/Client/init.lua:33]: Nope - try printing it while holding W and D for example.

**Comment by grilme99** [comment]
Closing this because the changes were worked in with the client refactor, but I had no good way of merging this in via Git.

**Pull request closed without merging**